docs/reference/commands/node-tool.asciidoc

```diff
@@ -0,0 +1,119 @@
+[[node-tool]]
+== elasticsearch-node
```

Comment:
We decided to move the docs changes to a separate PR - #37812, so this should go.